tests/projects.spec.ts

```diff
@@ -92,14 +92,9 @@ test("project edit configuration", async ({ page, lxdVersion }) => {
     .getByRole("navigation", { name: "Project form navigation" })
     .getByText("Networks")
     .click();
-  await setTextarea(
-    page,
-    "Available networks",
-    "Enter network names",
-    "lxcbr0",
-  );
-  await setTextarea(page, "Network uplinks", "Enter network names", "lxcbr0");
-  await setTextarea(page, "Network zones", "Enter network zones", "foo,bar");
+  await setTextarea(page, "Available networks", "Enter network names");
+  await setTextarea(page, "Network uplinks", "Enter network names");
+  await setTextarea(page, "Network zones", "Enter network zones");
```

Comment:
We removed the placeholder in the function definition (3rd argument), but here you remove the 4th argument. This should be changed, so we remove the 3rd argument instead.